content/applications/productivity/to_do.rst

```diff
@@ -0,0 +1,148 @@
+progress, and much more.
+
+Configuration
+=============
+
+If To-Do is not yet installed on your database, go to the :guilabel:`Apps`, search for it, and click
+:guilabel:`Install`.
+
+.. image:: to_do/install-to-do.png
+   :align: center
+   :alt: Install To-do app
+
+.. _to-do/personal-stages:
+
+Personal stages
+===============
+
+Your assigned to-dos are displayed on your dashboard in different by-default stages:
+
+- :guilabel:`Inbox`: new to-dos;
+- :guilabel:`Today`: to-dos of the day;
+- :guilabel:`This Week`: to-dos of the week;
+- :guilabel:`This Month`: to-dos of the month;
+- :guilabel:`Later`: to-dos planned for later;
+- :guilabel:`Done`: to-dos that are done;
+- :guilabel:`Canceled`: to-dos that have been canceled.
+
+.. note::
+   - Drag and drop a task to move it from one stage to another.
+   - Click the Settings button on a specific stage to :guilabel:`Fold`, :guilabel:`Edit`, or
+     :guilabel:`Delete` the stage.
+
+      .. image:: to_do/fold-edit-delete.png
+         :align: center
+         :alt: Settings button
+
+Creating to-dos
+===============
+
+To create a to-do, click :guilabel:`New` or the **+** button next to a stage name. A new card pops
+up. Add a title to your to-do, then click :guilabel:`Add` to save it, or :guilabel:`Edit` if you
+want to edit it.
+
+.. tip::
+   You can create a new to-do on the fly from any app in Odoo by clicking the clock icon, and
+   :guilabel:`Add a To-Do`:
+
+   .. image:: to_do/add-a-to-do-on-the-fly.png
+      :align: center
+      :alt: Add a to-do on the fly
+
+Editing to-dos
+==============
+
+Different actions are possible in a to-do:
+
+- :guilabel:`Mark as done`: click the check mark button when the task has been completed;
+- :guilabel:`Archive`, :guilabel:`Duplicate`, :guilabel:`Delete`, :guilabel:`Dashboard`, or
+  :guilabel:`Convert to task`: click on the cog icon to see these options;
+- :guilabel:`Tags`: select tags from the dropdown list to categorize your to-do. You can create a
+  new tag simply by writing it;
+- :guilabel:`Assignees`: select one or more users from the dropdown list to assign your to-do;
+- **Customize your tasks content**: when typing / in the text area, a dropdown list of features
+  related to the structure, format, media, navigation, widgets, banner, or basic blocks appear.
+- :ref:`Personal stages <to-do/personal-stages>`: move your task to another stage by clicking on
+  your desired stage.
+
+.. note::
+   To-dos, like private tasks, are only accessible to the users specified as assignees. You can use
+   the :guilabel:`Assignees` field to share a to-do with other users.
+
+.. image:: to_do/task-edition.png
+      :align: center
+      :alt: edit your task
+
+Scheduling activities
+=====================
+
+To help you get things done, you can schedule activities. From a to-do, click on
+:guilabel:`Activities`; or from the dashboard, click the clock icon on a to-do card, then
+:guilabel:`+ schedule an activity`:
```

Comment:
You can schedule activities from a to-do, click on Activities; or from the dashboard, click the clock icon on a to-do card, then + schedule an activity:

Reply:
updated a bit differently